src/routes/__tests__/vendor.test.tsx

```diff
@@ -1,6 +1,12 @@
 import { render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
 
+// import userEvent from '@testing-library/user-event'
+// import { http, HttpResponse } from 'msw'
+// import { API_BASE_URL } from '@/env.ts'
+// import { mswServer } from '@/lib/msw/index.ts'
```

Comment:
If we're going to comment out some code, please explain why we're commenting it out instead of removing it, for example: "For futute use when we integrate x y z"